Refit.Tests/RestService.cs

```diff
@@ -139,6 +139,24 @@ public interface ITrimTrailingForwardSlashApi
         Task Get();
     }
 
+    [RefitRestService("/api/Test")]
```

Comment:
Seems like this should be called `BaseAddressAttribute`?

Reply:
I renamed attribute